src/bootstrap.tsx

```diff
@@ -43,14 +50,62 @@ loadingMessage.textContent = 'Initializing Cytoscape. Please wait...'
 document.body.appendChild(loadingMessage)
 
 const keycloak = new Keycloak(keycloakConfig)
+
+const handleVerify = async () => {
+  await keycloak.loadUserProfile()
+  window.location.reload()
+}
+
+const handleCancel = () => {
+  keycloak.logout()
```

Comment:
I think there is a parameter here called redirectUri that is used in this logout function.  Depending on how cytoscape web is deployed e.g. test.ndexbio.org/cytoscape or web.cytsocape.org we might need to send a argument here.